### PR TITLE
Code review for latest changes

### PR DESCRIPTION
**Pull request by crezul**
Test changes

### CODE REVIEW

File: Tests/main.cpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #include "stdafx.h"
 
+
```
Comment (SirenkoAlexandr) [inline Tests/main.cpp]: Test commit

File: Tests/main.cpp [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
```
Comment (SirenkoAlexandr) [inline Tests/main.cpp]: 2nd  comment

File: Tests/main.cpp (hunk 1 above)
Comment (crezul) [reply Tests/main.cpp]: Ok

File: Tests/main.cpp (hunk 2 above)
Comment (crezul) [reply Tests/main.cpp]: ok2

**Comment by crezul** [comment]
finished

**Pull request merged**